docs/deployment/containers/docker.rst

```diff
@@ -405,46 +421,44 @@ You can do that by editing your `Docker Stack YAML file`_:
 Resource constraints
 ====================
 
-It is very important that you set resource constraints when you are running
-CrateDB inside Docker. Current versions of the JVM are unable to detect that
-they are running inside a container, and as a result, `the detected limits are
-wrong`_.
+It is important that you set resource constraints when you are running CrateDB
+inside Docker. Current versions of the JVM are unable to detect that they are
```

Comment:
This is a bit outdated. This was fixed at some point with JDK 8, and hasn't been a problem with JDK 11 (which we require since CrateDB 4.0).

I think we could remove this entire paragraph. Or maybe point out that users should work with cpu/memory limits if they plan to run multiple CrateDB containers on a single machine, to avoid CrateDB from over allocating resources